client/remote.go

```diff
@@ -188,3 +189,14 @@ func (remote *RemoteVPS) getDaemonAPIToken(session SSHSession, daemonVersion str
 	// There may be a newline, remove it.
 	return strings.TrimSuffix(stdout.String(), "\n"), nil
 }
+
+func (remote *RemoteVPS) SetRemoteProperty(name string, value string) bool {
+	f := reflect.ValueOf(remote).Elem().FieldByName(name)
+	if f.IsValid() {
```

Comment:
This code:
```go
	f := reflect.ValueOf(remote).Elem().FieldByName(name)
	if f.IsValid() {
		if f.Kind() == reflect.String {
			f.SetString(value)
			return true
		}
	}
	return false
```

Is repeated twice. It could probably be abstracted into a helper function that both `SetProperty` and `SetRemoteProperty` can rely on:

```go
func SetStructProperty(name string, value string, struct interface{}) { /* ... */ }
```